ovs/extensions/services/fleetctl.py

```diff
@@ -0,0 +1,237 @@
+# Copyright 2015 iNuron NV
+#
+# Licensed under the Open vStorage Modified Apache License (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.openvstorage.org/license
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Fleetctl module
+"""
+import time
+
+try:
+    import fleet.v1 as fleet
+    FLEET_CLIENT = fleet.Client('http+unix://%2Fvar%2Frun%2Ffleet.sock')
+except ImportError:
+    raise ImportError('Fleet python client is not installed. Please check documentation on how to install it')
+
+from ovs.log.logHandler import LogHandler
+from ovs.extensions.services.systemd import Systemd
+
+logger = LogHandler.get('extensions', name='fleetctl')
+
+
+class FleetCtl(object):
+    """
+    Contains all logic related to managing services through fleet
+    """
+
+    @staticmethod
+    def prepare_template(base_name, target_name, client):
+        return Systemd.prepare_template(base_name, target_name, client)
```

Comment:
Also defined in systemd and upstart - so defined here as well. Delegated to systemd since this is not a fleet specific action.